exec/jdbc-all/pom.xml

```diff
@@ -441,7 +441,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
                   
                   </message>
-                  <maxsize>20000000</maxsize>
+                  <maxsize>21000000</maxsize>
```

Comment:
@PythonicNinja can you make a  pass at adding the exclusions as well to this. The idea is to exclude any dependencies not directly needed for the CORS functionality.